Rendering/Tools/SplineInterpretationElement.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+	Copyright (c) 2020 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "SplineInterpretationElement.h"
+
+// PRIVATE VARIABLES
+// double length;
+// int indicator;
+// std::pair<size_t, size_t> indices;
+
+// CONSTRUCTOR
+OpenInfraPlatform::UserInterface::SplineInterpretationElement::SplineInterpretationElement() {}
+
+OpenInfraPlatform::UserInterface::SplineInterpretationElement::SplineInterpretationElement(
+	double _length, 
+	int _indicator, 
+	std::pair<size_t, size_t> _indices)
+{
+	length = _length;
+	indicator = _indicator;
+	indices = _indices;
+}
+
+
+// PUBLIC FUNCTIONS WITH ACCESS TO VARIABLES
+void OpenInfraPlatform::UserInterface::SplineInterpretationElement::setLength(double newLength) throw(...){
+	length = newLength; }

```

Comment:
Such a function does not throw, there is no `throw` statement within it. Use `throw()` to denote that.

Reply:
In commit b492fab, I removed the `throw` from the get- and set-functions.
Do you mean with "`throw` statement" something like the following lines?
```
if(badCase)
   throw oip::UnhandledException(myVariable);
```

Reply:
Yes, sth like this. Beware, though: if you use `throw()` as an indicator to a function definition, then neither the function itself or the functions it calls are not allowed to throw. Which means - if you are not throwing anything in the function and you do not call another function, where you are unsure if it throws, you can use the `throw()` indicator.